CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ set_target_properties(docopt PROPERTIES
   SOVERSION ${PROJECT_VERSION_MAJOR}
 )
 
-target_include_directories(docopt PUBLIC $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include/docopt> $<INSTALL_INTERFACE:include/docopt>)
+target_include_directories(docopt PUBLIC $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include> $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
```

Comment:
AFAICT this would change the include dir from `include/docopt` to just `include`, so this would not work.